lib/simple-seeder.js

```diff
@@ -223,18 +223,24 @@ module.exports = class SimpleSeeder {
   }
 
   async _createDrive (publicKey, info, opts = {}) {
-    const drive = new Hyperdrive(this.store.session(), publicKey)
-    await drive.ready()
-
-    this._suitCore(drive.core, info)
+    const core = this.store.get(publicKey)
+    await core.ready()
+    const blobs = this.store.get(Hyperdrive.getContentKey(core))
```

Comment:
Probably safer to stick with the existing order, so we already suit the core even if the blobs aren't available yet, and return before the blobs are ready (unsure if it matters in practice)

Something like
```
this._suitCore(core, info)
blobs.ready().then(() => this.suitCore(blobs)
```

Not quite sure what to do when the blobs.ready() errors though. I guess just swallowing it? (that's the equivalent of 'blobs' never being emitted, so drive.once waiting forever, which is the existing behaviour)